### PR TITLE
grcov 0.7.1 (new formula)

### PR DESCRIPTION
**Pull request by rajivshah3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/grcov.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      RUSTFLAGS:         rustflags,
+      RUSTDOCFLAGS:      "-Cpanic=abort",
+    }
+    system "cargo", "new", "hello_world", "--bin"
+    Dir.chdir(testpath/"hello_world")
+    with_env(env) do
+      system "cargo", "test"
+    end
+    system bin/"grcov", ".", "-t", "html"
+    assert_predicate testpath/"hello_world/html/index.html", :exist?
```
Comment (SMillerDev) [inline Formula/grcov.rb:29]:
```suggestion
    cd testpath/"hello_world" do
      with_env(env) do
        system "cargo", "test"
      end
      system bin/"grcov", ".", "-t", "html"
      assert_predicate testpath/"hello_world/html/index.html", :exist?
    end
```

Comment (rajivshah3) [reply Formula/grcov.rb]: Thanks!

File: Formula/grcov.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Grcov < Formula
+  desc "Rust tool to collect and aggregate code coverage data for multiple source files"
+  homepage "https://github.com/mozilla/grcov"
+  url "https://github.com/mozilla/grcov/archive/v0.7.1.tar.gz"
+  sha256 "cb7167ee8770d2903839e00d4a631cca08fb8095b2339c8796e7b9510d5f5055"
+  license "MPL-2.0"
+  head "https://github.com/mozilla/grcov.git"
+
+  depends_on "rust" => :build
```
Comment (carlocab) [inline Formula/grcov.rb:9]:
```suggestion
  depends_on "rust" => [:build, :test]
```
You need this to use `cargo` in the `test` block.

Comment (rajivshah3) [reply Formula/grcov.rb:9]: Ooh, thanks

**Comment by carlocab** [comment]
```
==> cargo test
   Compiling hello_world v0.1.0 (/private/tmp/grcov-test-20210125-40215-1aw07t1/hello_world)
error[E0463]: can't find crate for `profiler_builtins`
  |
  = note: the compiler may have been built without the profiler runtime

error: aborting due to previous error

For more information about this error, try `rustc --explain E0463`.
error: could not compile `hello_world`
```

**Pull request closed without merging**